### ISSUE TITLE
Wrapping in 'Shadow' dialog

### ISSUE THREAD

**Issue by spdegabrielle** [Metaxal/quickscript#49]
Wrapping in 'Shadow' dialog

![image](https://user-images.githubusercontent.com/108047/90198466-ab47dd00-ddc9-11ea-8da4-1eab1e87c470.png)

removing the newlines should  resolve.
https://github.com/Metaxal/quickscript/blob/d2cb292ec2c5d03c1735bcc0472beb153fb68a23/library-gui.rkt#L129

**Comment by spdegabrielle** [comment]
I'm also not a big fan of the DrRacket alert dialog with the tiny editor-canvas% and the 'copy text' context menu but I'm pretty sure that is inherited from DrRacket.

**Comment by Metaxal** [comment]
Good point. Probably best to replace this with an ad-hoc editor-canvas%.

**Comment by spdegabrielle** [comment]
I was more thinking of moving it to the strings library to support languages as that supports text without unfortunate line-breaks.
Fixing the DrRacket alert mechanism is a separate issue.

**Comment by spdegabrielle** [comment]
Would this be an acceptable interim fix: 

https://docs.racket-lang.org/scribble/reader.html?#%28part._.The_.Command_.Part%29
```
@foo{bar @; comment
     baz@;
     blah}
```


```
(message-box "Create shadow script?"
                            @string-append{
 This will:

 1) Disable the script file
 @(path->string (build-path dir file))

 2) Create a new 'shadow' script file
 @(path->string new-script-path)
 that has the same behaviour as the original script file but can be modified @;
 to change its properties;
 @(if drracket-parent?
      "\n3) Open the shadow script file for edition in DrRacket."
      "")
   
 This allows the original script file to be updated (e.g., if part of a package) @;
 while allowing the user (you) to change its properties without changing the @;
 original file directly.

 Do you want to proceed?
}
                            fr
                            '(caution yes-no))))

```

**Comment by Metaxal** [comment]
I guess that would be fine, yes. Thanks! (will do later, or you can make a PR if you like)

**Comment by Metaxal** [comment]
Fixed by eb298b7433ffdc8650ff92f2f9e1d93af9d8165a
Thank Stephen!
